docs/walkthrough.md

```diff
@@ -53,7 +53,14 @@ the appropriate values will be automatically populated.
 However, if you do not see the `requestheader-client-ca-file` value in the
 config map (which is the case on some GKE setups), you can either populate
 the configmap with a valid CA certificate, or set the
-`apiserver.tls.requestHeaderCA` option when installing the Helm chart.
+`apiserver.tls.requestHeaderCA` option when installing the Helm chart:
+
+```shell
+helm install charts/catalog --name catalog --namespace catalog --set apiserver.tls.requestHeaderCA=$(base64 --wrap 0 /path/to/requestheader-client-ca.crt)
```

Comment:
2 things:

1. We need a series of CLI commands in the walk-through to explain how to create one of these CAs, separate from the `auth.md` file
2. The auth.md file is great for more details, but from where do we link it?